### ISSUE TITLE
the equation for daze 

### ISSUE THREAD

**Issue by 5m1l35** [yhrcyt/StS-TheWarriorMod#12]
it can be rewritten as 70.7106781187/sqrt(average speed of cards played) 
x=number of cards y= total speed 
current z=sqrt(x*5000/y) correct
mean average would be y/x
and obviously x(y/x)=y
sub z=sqrt(x*5000/x(y/x))
z=sprt(5000/(y/x))
z=70.7106781187/sqrt(y/x) This to me is alot easier to understand we are only dealing with values over 0 so this operation is valid

**Comment by 5m1l35** [comment]
you could also sqrt all the speed values before hand and make it simpilar to calculate

**Comment by yhrcyt** [comment]
but you also need to sqrt number of cards

**Comment by 5m1l35** [comment]
yes you sould but the square root of 1 and 4 is pretty easy to calculate. in your head it also makes it a little easier to kind of get an idea of how much daze you are going to get idk maybe its just me.

**Comment by yhrcyt** [comment]
You also have to multiply the damage monster is doing by (100-dazed amount)/100, while dazed stack based on percentage... I don't wanna simply change the description, that's not the solution.

**Comment by yhrcyt** [comment]
I wanna make the player somehow preview everything a combo does.

**Comment by 5m1l35** [comment]
I forgot about that good point maybe have like a power variable under the player that shows where the combo is currently at teling how much daze it will give its the best I can come up with

**Comment by 5m1l35** [comment]
Tho I really dont know how the game works under the hood so that may be a really bad idea.

**Comment by 5m1l35** [comment]
or maybe do something with the speach bubbles like he is thinking

**Comment by 5m1l35** [comment]
I would imagine those just take a string

**Comment by yhrcyt** [comment]
But when card combo, it opens grid select screen which pauses other thing.

**Comment by 5m1l35** [comment]
ohh good point

**Comment by 5m1l35** [comment]
could you put like a combo card at the side of the screen that maintains with like damage daze amount and  distraction because I saw that thrust and cut modified their damage values based on what happened before and I know that you can call other variables into that screen even if the game does not update in the background. Tho that does sound like a ridiculous amount of work so I don't know if you would be willing to do that. Honestly I would have to better understand how you setup that screen before I could even really understand how to do this.

**Comment by yhrcyt** [comment]
how about when you finished a combo, don't do the actual thing but just visually display what you've done in this combo, and give you a chance to cancel the entire combo

that seems hard but it's definitely doable

**Comment by 5m1l35** [comment]
That would be interesting very different.

**Comment by 5m1l35** [comment]
Tho it does sound like a head ache and a half

**Comment by yhrcyt** [comment]
why I've designed this mod
I made it so complicated that I ran out off idea super fast and had to deal a LOT of technical issues
all attack cards are almost exactly the same

by the way do you have some more ideas to make those cards more diverse and less repetitive
it's 99% repetitive right now

**Comment by yhrcyt** [comment]
I'm currently working on a mod called "Ascension Minus Mod" that can let your asc go negative. It attracts WAY much more attention than this mod did, so if there's no bug to fix, I'm gonna finish that mod first.

**Comment by 5m1l35** [comment]
imagination has never realy been my strength honestly I don't actually think the cards are any more repetitive then the base games ideas tho

**Comment by 5m1l35** [comment]
fair

**Comment by yhrcyt** [comment]
it's the community feedback.
all cards with same attack type do almost the same thing with only a little number change.

**Comment by 5m1l35** [comment]
That just made thematic sense to me the guy has a range of attacks he can do with a weapon type  independent of how the weapon is shaped he will do about the same damage being the same guy but it will have varied secondary effects.

**Comment by yhrcyt** [comment]
But I still think the cards are too similar. I don't even have a card that does multiple times of damage. And also shouldn't we have a claw that cannot grasp or a sword that can disarm? There're so many possibilities, the design can definitely be improved to be more diverse.

And also THIS MOD DOESN'T EVEN HAVE RELICS shouldn't we design a few?

**Comment by 5m1l35** [comment]
maybe cards to start a combo but are also defensive like smoke bomb which can be followed with a scratch cut or feint attack and gives you a evasion chance(would proably set that up to on hit before application rng then increase block to amount of attack if true), battle rage which can be followed by blow hammer or strike attack and gives 30% reduced damage on the player. into the fray which is like you are steping into the middle of everything and gives slash, strike attacks 50% aoe to enemies adjacent to your target(have no idea how you would code this) maybe also make it so each one of those buffs the attacks that are those types by some amount. BTW I havn't actually finished unlocking all the cards so if these already exist I am sorry for being a idiot. These also all sound like a balancing nightmare.

**Comment by yhrcyt** [comment]
What do yo think of more weapon types? e.g. a weapon type that can Grasp, Slash or Blow.

I'm gonna release a BETA version soon, so I'm planing to add a lot of content.
I want to change the current "Grasped" power name to "Nerfed" so other attack types can use it.
And I need to give the starting relic a name. What should I name it?

I'm planning to add a list of played card in a combo on the left of the screen. You can view every card you've played. The "Combo Card" as an integration you've mentioned before is not possible (at least very difficult).

I just realized you watched this repository. You opened lots of new issues. Thank you so much for supporting. When beta release, you will see your name in "Credits".

**Comment by 5m1l35** [comment]
Yeah I figured it would not be plausible.  Interesting type idea so basically it ends combos. Interesting I am honestly not sure blow fits on it unless it is going to have a similar cost to hammer.  Because the only time you use blow is either to get lethal or as a stand alone play. What weapon would this be because I cant think of something big that can really grasp like a pole arm fits pretty well for the other two but I don't see it with grasp I mean that is fine I was never quite sure what that meant anyway. As for the name of the relic its a bit hard. The class can be very brutish but the relic is very strategic in effect if you know what I am saying. I have the name Gladiatoria from a old fencing manual that I think makes sense if you change the art to be Enchiridian with maybe palate switching to a darker shade. was also thinking about relics like one could be quick draw sheath that allows you to retain one of your weapon cards with the idea of allowing you to setup a better combo. This could also maybe be turned into a power.  Maybe another manual Arte of Defense that makes it so any damage you block is doubled and dealt back to the attacker making the shield cards more useful. BTW oricalcum is real op with this class.
